src/plugins/es_ui_shared/__packages_do_not_import__/ace/use_ui_ace_keyboard_mode.tsx

```diff
@@ -9,23 +9,32 @@
 import React, { useEffect, useRef } from 'react';
 import * as ReactDOM from 'react-dom';
 import { keys, EuiText } from '@elastic/eui';
+import { KibanaRenderContextProvider } from '@kbn/react-kibana-context-render';
 
 import './_ui_ace_keyboard_mode.scss';
+import type { AnalyticsServiceStart, I18nStart, ThemeServiceStart } from '@kbn/core/public';
 
-const OverlayText = () => (
+interface StartServices {
+  analytics: Pick<AnalyticsServiceStart, 'reportEvent'>;
+  i18n: I18nStart;
+  theme: Pick<ThemeServiceStart, 'theme$'>;
+}
+
+const OverlayText = (startServices: StartServices) => (
   // The point of this element is for accessibility purposes, so ignore eslint error
   // in this case
   //
-  <>
+  <KibanaRenderContextProvider {...startServices}>
```

Comment:
This is the change that directly addresses https://github.com/elastic/kibana/pull/180819